model/testing/src/icon4py/model/testing/datatest_fixtures.py

```diff
@@ -79,6 +79,8 @@ def download_ser_data(request, processor_props, ranked_data_path, experiment, py
 @pytest.fixture
 def data_provider(download_ser_data, ranked_data_path, experiment, processor_props):
     data_path = dt_utils.get_datapath_for_experiment(ranked_data_path, experiment)
+    print(ranked_data_path)
```

Comment:
I assume that is for debugging failing tests.

Reply:
Yes. Fixed.